src/components/CostEstimationContent/CostHeader.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FunctionComponent, useEffect, useState } from "react";
+import { FiatLabel } from "@govtechsg/open-attestation-utils";
+import { useFetchGasPrice } from "../../common/hooks/useFetchGasPrice";
+import { currentDateStr } from "../../utils";
+
+export const CostHeader: FunctionComponent = () => {
+  const [dateTime, setDateTime] = useState(currentDateStr());
+  const { price, gwei } = useFetchGasPrice("ethereum", 30000);
+  const { price: maticPrice, gwei: maticGwei } = useFetchGasPrice("polygon", 30000);
+  const priceFactor = gwei * 0.000000001 * price;
+  const maticPriceFactor = maticGwei * 0.000000001 * maticPrice;
+
+  const defaultGasLimit = 43634;
```

Comment:
inidicate that this is the estimated gas cost for xfer of holder instead of just `defaultGasLimit`

Reply:
ok